user/pages/03.commerce2/03.user-guide/05.payments/03.capture/docs.md

```diff
@@ -1,7 +1,21 @@
 ---
-title: Capture and authorization
+title: Capture an authorization
 taxonomy:
     category: docs
 ---
 
-! We need help filling out this section! Feel free to follow the *edit this page* link and contribute.
+Payments can be authorized during the checkout flow and captured separately by administrative users at a later time (e.g., when a shipment is sent to fulfillment, when a shipment is shipped, or when a shipment is delivered). Note: not all payment gateways support this feature. Store administrators will only have the ability to capture a prior authorization if the payment gateway was configured for this type of transaction at the time the order was placed.
+
+To capture a prior payment authorization for an order navigate to the **Orders** management page from the **Commerce** menu using the administrative toolbar. Search or browse for the order with an authorization to be captured. Expand the operations popup and click **Payments** from the menu or click the hyperlinked order number to open the order view page then click the **Payments** tab to view the order payments.

```

Comment:
Missing comma: 
To capture a prior payment authorization for an order, navigate to the...
Run-on sentence: 
... to open the order view page. Then click the...

Reply:
Corrected.